CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Contributor guidelines. [#574](https://github.com/rokwire/events-manager/issues/574)
 - A pull request template. [#575](https://github.com/rokwire/events-manager/issues/575)
 - Add detect-secrets.yaml. [#591](https://github.com/rokwire/events-manager/issues/591)
+- Refresh setting page after adding a new calendar. [#605](https://github.com/rokwire/events-manager/issues/605)
```

Comment:
Please merge the latest develop branch changes to this branch and move the CHANGELOG entry under the unreleased section. The rest looks good. Thanks.